maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/scopes/MavenDependencyScopes.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.repository.internal.scopes;
+
+/**
+ * The dependency scopes used for Java dependencies in Maven. This class defines labels only, that are doing pass-thru
+ * over Resolver.
+ *
+ * @since 4.0.0
+ */
+public final class MavenDependencyScopes {
```

Comment:
Should that be part of the Maven 4 API ?

Reply:
Or is that for a follow-up PR, maybe it needs other adjustements...

Reply:
Yes, this PR merely "sets the baseline": all is as before, but code is now part of Maven codebase (not blurred between Resolver and Maven).

Reply:
Yes, when the followup PR ultimate goal is reached, this class MavenDependencyScopes should be gone